OpenRA.Mods.RA/Traits/Chronoshiftable.cs

```diff
@@ -41,6 +49,7 @@ public Chronoshiftable(ActorInitializer init, ChronoshiftableInfo info)
 		{
 			this.info = info;
 			self = init.Self;
+			iPositional = self.TraitOrDefault<IPositionable>();
```

Comment:
Please implement `INotifyCreated` and move this there to avoid trait-initialization-order bugs.